music/app/controllers/services_controller.rb

```diff
@@ -11,7 +11,9 @@ def new
   def create
     @resource = OpenStruct.new(date: Date.parse(params[:resource][:date]), body: params[:resource][:body])
```

Comment:
"If for some unfathomable reason you want to..."

Those are daaaangerous words...